Media print rules added in CSS

jrchudy opened the pull request:
This PR addresses issues #1381 and #1383.

`@media print {...}` was added to change some of the styling when print is used from the browser tools. This should make printing documents look better than before and remove the limitation that bound it to just the first page. 

Extra padding was added to the main container so that the `Loading...` text shows up in a better place now.

RFSH commented:
Is there a reason for `#rt-loading` span to be outside the `main-container`? You can just move it up inside the `main-container` and then remove the extra padding that you added.

jrchudy commented:
@RFSH I moved the "Loading..." div to inside the main container. This extended the space of that container and worked as you suggested, but the text was cut off by the footer so I added another `10 px` of padding.
